src/components/Demo/constants/config-mock.js

```diff
@@ -25,10 +25,12 @@ const CONFIG_MOCK = {
   scaleType: 'zoom',
 
   colorTheme: {
+    seatMapBackgroundColor: '#fff',
```

Comment:
need to use `seatMapBackgroundColor` as background, not only for wings' leadings